### DISCUSSION TITLE
[Suggestion] Make Timestamps and Soft Delete timestamp Fields Viewable

### DISCUSSION THREAD

**Discussion by relayer464** [TiagoSilvaPereira/vemto-issues#208] [Ideas & Suggestions]
Greetings,

I really like how you can check a box to add "Has Timestamps" and/or "Uses Soft-Deletes" to your Entity on the Schema Editor and the created_at, updated_at, and deleted_at fields are added to your database.  However, when I switch over to the CRUD editor I don't see where any of those fields are available to add to your Blade pages?  I understand it doesn't make much sense to make them available for the "On Create" or "On Update", but it would be nice if they were available on the CRUD editor so they could be added to the "On Show" and/or "On Index" pages.   Perhaps I've overlooked an option?  If not, it would be nice to have this option.  Just a suggestion. 

Thanks!

**Comment by TiagoSilvaPereira** [comment]
Thank you for your suggestions @relayer464
